src/tox/plugin/manager.py

```diff
@@ -42,12 +43,16 @@ def __init__(self) -> None:
         )
 
         for plugin in internal_plugins:
+            logging.info("Registering %s", plugin)
             self.manager.register(plugin)
-        self.manager.load_setuptools_entrypoints(NAME)
-        self.manager.register(state)
+        try:
+            self.manager.load_setuptools_entrypoints(NAME)
+            self.manager.register(state)
 
-        REGISTER.populate(self)
-        self.manager.check_pending()
+            REGISTER.populate(self)
+            self.manager.check_pending()
+        except ImportError as e:
+            logging.warning("Ignored plugin %s which failed to load. %s", NAME, e)
```

Comment:
Here you want `%r` to show the type too. And ideally, we'd also show the last source code line of the traceaback (you can get it via traceback.format_exc).